x-pack/filebeat/module/zeek/connection/config/connection.yml

```diff
@@ -53,10 +53,29 @@ processors:
           to: "destination.packets"
 
         - from: "zeek.connection.conn_state"
-          to: "zeek.connection.state"
-          
+          to: "zeek.connection.state.code"
```

Comment:
This is breaking change to the mapping. We need to call this out in the changelog and take action to not affect 7.x with this change. @alakahakai can you look into fixing the 7.x branch before the 7.6 branch gets created.

Reply:
PR [#15169](https://github.com/elastic/beats/pull/15169)